### PR TITLE
Allow building with(out) Lomiri features

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -65,12 +66,17 @@ find_package(PkgConfig REQUIRED)
 
 # glib...
 set(GLIB_MINIMUM 2.36)
-set (SERVICE_DEPS libayatana-common>=0.9.3 gio-unix-2.0>=${GLIB_MINIMUM} glib-2.0>=${GLIB_MINIMUM} gudev-1.0)
+set (SERVICE_DEPS libayatana-common>=0.9.3 gio-unix-2.0>=${GLIB_MINIMUM} glib-2.0>=${GLIB_MINIMUM})
 
 if (ENABLE_TESTS)
     list (APPEND SERVICE_DEPS properties-cpp>=0.0.1)
 endif ()
```
Comment (sunweaver) [inline CMakeLists.txt:73]: I think that the above part needs to be addressed, too, for another reason. If I look at the code, I see that core/signal.h and core/property.h gets included in:

```
[sunweaver@sunobo ayatana-indicator-display (main)]$ grep -rE "(property.h|signal.h)"
tests/utils/glib-fixture.h:      g_signal_handler_disconnect(o, handler_id);
src/adbd-client.h:#include <core/signal.h>
src/usb-snap.h:#include <core/signal.h>
src/usb-monitor.h:#include <core/signal.h>
src/greeter.h:#include <core/property.h>
src/rotation-lock.cpp:    g_signal_handlers_disconnect_by_data(m_settings, this);
src/exporter.h:#include <core/signal.h>
src/usb-monitor.cpp:        g_signal_handlers_disconnect_by_data(m_udev_client, this);
src/indicator.h:#include <core/property.h>
```

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -4,18 +4,13 @@ add_compile_options(
...
-    adbd-client.cpp
-    exporter.cpp
-    greeter.cpp
-    indicator.cpp
-    rotation-lock.cpp
-    usb-manager.cpp
-    usb-monitor.cpp
-    usb-snap.cpp
-)
+set (SERVICE_LIB_SOURCES exporter.cpp indicator.cpp rotation-lock.cpp)
```
Comment (sunweaver) [inline src/CMakeLists.txt:7]: why not keep this multi-line?

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -4,18 +4,13 @@ add_compile_options(
...
+
+if (ENABLE_LOMIRI_FEATURES)
+    list (APPEND SERVICE_LIB_SOURCES greeter.cpp adbd-client.cpp usb-manager.cpp usb-monitor.cpp usb-snap.cpp)
```
Comment (sunweaver) [inline src/CMakeLists.txt:10]: multi-line here, too. It's much better readable then, I guess.

**Review by sunweaver — CHANGES_REQUESTED** [review]
Some minor changes required. Please check.

File: src/CMakeLists.txt (hunk 2 above)
Comment (tari01) [reply src/CMakeLists.txt]: Amended

File: src/CMakeLists.txt (hunk 3 above)
Comment (tari01) [reply src/CMakeLists.txt]: Amended

File: CMakeLists.txt (hunk 1 above)
Comment (tari01) [reply CMakeLists.txt:73]: Amended CMakeLists.txt and src/CMakeLists.txt

**Comment by JamiKettunen** [comment]
Ping @sunweaver :)

**Pull request merged**